utility/src/main/java/org/oppia/util/logging/EventLogger.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.util.logging
+
+import android.content.Context
+import org.oppia.app.model.EventLog
+
+/**
+ * Logger for tracking events.
+ * Note that this utility may later upload them to remote services
+ * or log them to a file on disk.
+ */
+interface EventLogger {
+
+  /**
+   * Logs events to remote services or log them to a file on disk.
```

Comment:
This interface is not meant to store them to a file, thats what the controller is supposed to do right?

Reply:
Yeah, will change that. Actually this was suggested to me by Ben in the CrashLogger PR, so I just placed it here as well.

Reply:
Done.